client/build.sbt

```diff
@@ -41,7 +41,7 @@ credentials += Credentials(
   sys.env.getOrElse("SONATYPE_PASS", "")
 )
 
-isSnapshot := version.value endsWith "SNAPSHOT"
+isSnapshot := true
```

Comment:
Due to this problem: https://stackoverflow.com/questions/27530507/sbt-publish-only-when-version-does-not-exist